tensorrt_llm/_torch/memory_buffers_util.py

```
@@ -0,0 +1,63 @@
+class Buffers:
+
+    def __init__(self):
+        buffers: {str, list[(torch.Tensor, bool)]} = {}
```

Ruff (0.12.2)

Local variable `buffers` is assigned to but never used

Remove assignment to unused variable `buffers`

(F841)